test_data/strain/perfect_solution/expected_analysis.json

```diff
@@ -0,0 +1 @@
+{"summary":"No suggestions found.","comments":[]}
```

Comment:
I'm wondering if the student benefits from having this summary being displayed, as just having

```suggestion
{"comments":[]}
```

would have the website show that there are no suggestions. I think I would prefer the latter.

Reply:
I'm pretty sure that when there are no comments, the website doesn't display anything.
The elixir analyzer exports the exact same file and nothing is shown.
I'd prefer having the summary there, even if it isn't displayed, it makes the code simpler and the analysis file simpler to understand from a dev point of view.

![Screen Shot 2022-03-02 at 10 42 03](https://user-images.githubusercontent.com/16929078/156278552-894dca5d-1ff0-48b0-a98e-5429b06e33f1.png)